app/controllers/tag_controller.rb

```diff
@@ -73,15 +73,6 @@ def index
   end
 
   def show
-
-    # Enhancement #6306 - Add counts to `by type` dropdown on tag pages
-    @counts = {:posts => 0, :questions => 0, :wiki => 0 }
-    @counts[:posts] = Tag.find_nodes_by_type([params[:id]], 'note', false).count
-    @counts[:questions] = Tag.find_nodes_by_type("question:#{params[:id]}", 'note', false).count
-    @counts[:wiki] = Tag.find_nodes_by_type([params[:id]], 'page', false).count
-    params[:counts] = @counts
-    # end Enhancement #6306 ============================================
```

Comment:
I fixed these counts and moved them to a private method in this same controller.